bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/dsf/LaunchConfigurationDelegate.java

```diff
@@ -299,33 +290,13 @@ public void launch(ILaunchConfiguration config, String mode, ILaunch launch, IPr
 		{
 			monitor = new NullProgressMonitor();
 		}
-		if (config.getAttribute(ICDTLaunchConfigurationConstants.ATTR_PROGRAM_NAME,
-				DefaultPreferences.PROGRAM_APP_DEFAULT).contentEquals(DefaultPreferences.PROGRAM_APP_DEFAULT))
-		{
-			setProgramNameAtr(config);
-		}
```

Comment:
removed this because it violates SRP and we no longer need this workaround. The correct binary path will be determined by the DefaultAppResolver.